client/pkg/utils/grpcutil/grpcutil.go

```diff
@@ -41,6 +42,53 @@ const (
 	FollowerHandleMetadataKey = "pd-allow-follower-handle"
 )
 
+// Add retry related CallOption
+type retryCallOption struct {
+	grpc.EmptyCallOption
```

Comment:
why need this?

Reply:
Otherwise, we need to implement before and after interfaces.